### PR TITLE
Created contact form

### PR DESCRIPTION
**Pull request by shinyenyi**
Implemented html code and css styling to form contact and footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -180,7 +180,35 @@ <h1>Skills</h1>
     </ul>
 
   </section>
-
+  <section class="contact-form">
+    <form class="my-form" action="https://formspree.io/f/mrgdbqpg" method="POST">
+      <p class="contact-form-content">I'm always interested in hearing about new projects, so if you'd like to chat
+        please get in touch.</p>
+      <input class="contact-form-name" type="text" name="name" placeholder="Full name">
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to add all validations as requested in the project requirements. Please add the `required` attribute which helps us to not submit empty form input. Also, add `maxlength="30"` validation to validate the maximum input character to be inserted in the name `input`. :wink:

File: index.html [hunk 2]
```diff
@@ -180,7 +180,35 @@ <h1>Skills</h1>
...
+      <input class="contact-form-email" type="email" name="email" placeholder="Email address">
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to add all validations as requested in the project requirements. Please add the `required` attribute which helps us not submit empty form input.

File: index.html [hunk 3]
```diff
@@ -180,7 +180,35 @@ <h1>Skills</h1>
...
+      <textarea class="contact-form-message" rows="3" name="message" placeholder="Enter text here"></textarea>
```
Comment (levy002) [inline index.html]: - [ ] Kindly add the validation for the maximum character for this `textarea` to `maxlength="500"`.:wink:

File: styles.css [hunk 4]
```diff
@@ -533,3 +533,117 @@ button[enabled] {
...
+  padding: 20px;
+  margin-bottom: 24px;
+}
+
+.contact-form-message {
+  margin-bottom: 24px;
+  border: none;
+  padding: 20px;
+  background-color: #fbf8f7;
+}
```
Comment (levy002) [inline styles.css]: - [ ] Kindly adjust the width of the `textarea` in order to meet the provided design. :wink: 

   - ### Your design
  
   image


    - ### Figma design
   image

File: styles.css [hunk 5]
```diff
@@ -533,3 +533,117 @@ button[enabled] {
...
+  margin-top: 108px;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  margin-bottom: 70px;
+}
```
Comment (levy002) [inline styles.css]: - [ ] Kindly align your paragraph correctly in order to look exactly like the provided design. :wink: 

   - ### Your design
     image


  - ### Figma design
   image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hi @shinyenyi  && @Khaled-AlKhateeb  ,  :wave: 
### I'm Levy, your PR reviewer

Excellent, you did a great job :+1:  
However, they are some issues that you need to address before you move on to the next project :wink:

## Highlights
- Good job for using the correct git-flow :heavy_check_mark: 
- All linters are passing :heavy_check_mark: 
### Status: REQUESTED CHANGES :recycle: 
_check my comments under review_
 -  [ ] I think it would be better if we add all contributors' information to your README file. :wink: 
 - [ ] Kindly make sure to add a good  PR description. It makes your repo looks more professional. wink

  
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] On your readme, you have the section "license" according to the [Microverse readme template](https://github.com/microverseinc/readme-template).  Kindly, make sure that The link featured in that section should points to a file at the root of your project (the license file).   Just copy it from the mentioned template:

   image

That way, you will have a professional repo, and your project will be well documented 💪 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @levy002 in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ghost — APPROVED** [review]
Hi @shinyenyi , 

Good work implementing the changes requested. Your project looks great!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)
## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good Design👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**